pkg/workflows/wasm/runner_wasip1.go

```diff
@@ -51,7 +58,37 @@ func NewRunner() *Runner {
 			os.Exit(code)
 		},
 		SDK: Runtime{
-			Logger: logger.NewWithSync(&wasmWriteSyncer{}),
+			Logger: l,
+			Fetch: func(req TargetRequestPayload) TargetResponsePayload {
+
+				b, err := json.Marshal(req)
+				if err != nil {
+					os.Exit(CodeRunnerErr)
+				}
+				reqptr, reqptrlen := bufferToPointerLen(b)
+
+				respBuffer := make([]byte, 1024*5)
```

Comment:
We should try to make this configurable from the host somehow, probably by including it in the Compute request args. As it stands this will be hardcoded in any binaries which could be painful if a binary exceeds this limit. By passing it from the host it'll at least be configurable even if we just hardcode it there 🙈